packages/theming/win32-theme/src/NativeModule/getThemingModule.native.ts

```diff
@@ -1,28 +1,31 @@
 import { OfficeThemingModule } from './officeThemingModule';
-import { NativeEventEmitter } from 'react-native';
+import { NativeEventEmitter, TurboModuleRegistry, EventSubscriptionVendor, TurboModule } from 'react-native';
 import { fallbackGetPalette, fallbackOfficeModule } from './fallbackOfficeModule';
 
-declare module 'react-native' {
-  interface ITurboModuleRegistry {
-    get: (name: 'Theming') => OfficeThemingModule & EventSubscriptionVendor;
-  }
-  const TurboModuleRegistry: ITurboModuleRegistry;
-}
+// declare module 'react-native' {
+//   interface ITurboModuleRegistry {
+//     get: (name: 'Theming') => OfficeThemingModule & EventSubscriptionVendor;
+//   }
+//   const TurboModuleRegistry: ITurboModuleRegistry;
+// }
 
-import { TurboModuleRegistry } from 'react-native';
+// import { TurboModuleRegistry } from 'react-native';
 
 /**
  *  If we have a userAgent string, let's assume we're web debugging.  __DEV__ is for developer bundles.  Currently,
  *  react-native only polyfills navigator with { product: 'ReactNative', geolocation: NativeModules.Geolocation }
  */
+
+type ITurboModuleRegistry = TurboModule & OfficeThemingModule & EventSubscriptionVendor;
+
 function disableGetPalette(): boolean {
   const disabled = __DEV__ && navigator && navigator.userAgent !== undefined;
   disabled && console.warn(console.warn('Web Debugging forces Theming Native Module to fallback to fake color values.'));
   return disabled;
 }
 
 export function getThemingModule(): [OfficeThemingModule, NativeEventEmitter | undefined] {
-  const module = TurboModuleRegistry.get('Theming');
+  const module = TurboModuleRegistry.get<ITurboModuleRegistry>('Theming');
```

Comment:
Out of curiosity have you checked that colors from Office are working properly on win32? You can follow the steps here to check: https://github.com/microsoft/fluentui-react-native/blob/master/docs/pages/Testing/TestingThemesWin32.md

Reply:
I haven't, I'll checkout the steps, thanks

Reply:
I checked it and I believe it's working
![image](https://user-images.githubusercontent.com/20259210/178360428-0573ab01-fb7e-49da-b907-acff3d3aa25d.png)